src/vaults/mainnet.json

```
@@ -10,6 +10,11 @@
       "slug": "beraborrow-bwbera",
       "stakeTokenAddress": "0x6751C09113a83a83a43829fD0b3BC0D7bdbe07bf"
     },
+    {
+      "beraRewardsVault": "0x1161e6a6600C08c21cFF7Ac689e781b41Db56d85",
+      "slug": "beraborrow-snect",
```

slug has to follow protocol-token.name
```suggestion
      "slug": "beraborrow-staked-nectar",
```